src/lib/util.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 var $ = require('$jquery');
```

Comment:
You're absolutely right - this is ripe for refactor, but I'll save that for another PR (not my code 😜 )